### ISSUE TITLE
flip_left_right True causes retrain.py to hand in the midle of the training

### ISSUE THREAD

**Issue by snarb** [tensorflow/hub#91]
Tested on the 20k * 3 classes dataset. Without flip_left_right True all works fine. With flip_left_right True training is hanging in the process. Last message "INFO:tensorflow:2018-06-27 14:23:49.600421: Step 310: Train accuracy = 78.0%". Tested several times.

**Comment by akhorlin** [comment]
@arnoegw might have a suggestion here

If I understand the situation correctly, training (and eval?) runs successfully for some number of steps, then the training terminates? It's a bit hard to say what is the issue here without knowing additional details. What is your training setup (os, py version, gpu/cpu, etc)? How does resource usage looks during training, how much memory/cpu is being used?  How does cpu/memory usage looks like as the training progresses?

**Comment by snarb** [comment]
Yes, training is running successfully for some number of step(different sometimes 310 sometimes 40). This is ubuntu 16. Tf 1.8 inside the Jupiter. Master version of the hub I have dropped this setting and have no problems now. Just a feedback. I think some kind of leak.

**Comment by akhorlin** [comment]
That (memory leak) was my initial guess as well. One option to validate is
try to profile using TF profiler (
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/profiler/README.md)
and if it is indeed a memory leak in some TF op then file a bug report with
TF.

**Comment by andresusanopinto** [comment]
Closing as innactive.
